webapp/app/assessment/fra/sections/originalDataPoint/actions/updateSectionTables.js

```diff
@@ -88,3 +88,54 @@ export const getUpdateTablesWithOdp = (state, odp, draft = true) => {
   actions.push(getUpdateForestCharacteristics(state, odp, draft))
   return actions.filter((action) => !!action)
 }
+
+const getUpdateSectionTableNoNDP = (state, sectionName, tableName, year) => {
```

Comment:
this function is the same than ```getUpdateSectionTable``` with the difference of the object passed (odpDatum). pass the datumOdp to getUpdateSectionTable instead